modules/users/server/config/strategies/twitter.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
 			var providerUserProfile = {
 				displayName: profile.displayName,
 				username: profile.username,
-				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value : undefined,
+				profileImageURL: (profile.photos && profile.photos.length) ? profile.photos[0].value.replace('normal', 'bigger') : undefined,
```

Comment:
Is there a case where a user won't have a photo that you have to check that it exists first? I would think even the bird would be the first profile picture, no? And is [0] the latest or the first?